### PR TITLE
run provider generator to sync changes from core

### PR DESCRIPTION
**Pull request by durandom**
ran the generator again with

```bash
bundle exec rails g provider <name> --path ../
```

these are the changes.

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  :ems_lenovo:
+    :blacklisted_event_names: []
+    :event_handling:
+      :event_groups:
+:http_proxy:
+  :lenovo:
+    :host:
+    :password:
+    :port:
+    :user:
```
Comment (Fryguy) [inline config/settings.yml]: Are there any settings that have to come over from ManageIQ?

Comment (durandom) [inline config/settings.yml]: yes there are some, but should be part of a different PR

Comment (durandom) [inline config/settings.yml]: created https://github.com/ManageIQ/manageiq-providers-lenovo/issues/65
